tests/publiccloud/run_ltp.pm

```diff
@@ -90,7 +130,20 @@ sub run {
         $instance->run_ssh_command(cmd => 'sudo zypper -q in -y ltp',                              timeout => 600);
     }
 
-    $ver_linux_log = $instance->run_ssh_command(cmd => get_ltproot() . "/ver_linux");
+    download_whitelist();
+    my $ltp_env = gen_ltp_env($instance);
+    $self->{ltp_env} = $ltp_env;
+
+    # Use lib/LTP/WhiteList module to exclude tests
+    if (get_var('LTP_KNOWN_ISSUES')) {
+        my $exclude       = get_required_var('COMMAND_EXCLUDE');
```

Comment:
Why is this var required?

Reply:
Because we tell it is required later in the script as well. But not needed, thx!